package/libnsh/libnsh.mk

```diff
@@ -0,0 +1,19 @@
+################################################################################
+#
+# libnsh
+#
+################################################################################
+
+LIBNSH_VERSION = 0.4
+LIBNSH_PKGNAME = libnsh
+LIBNSH_SOURCE = $(LIBNSH_PKGNAME)-$(LIBNSH_VERSION).tar.gz
+# This looks like a typo, but the 'v' needs to be there
+LIBNSH_SITE = https://github.com/westermo/$(LIBNSH_PKGNAME)/releases/download/v$(LIBNSH_VERSION)
+LIBNSH_INSTALL_STAGING = YES
+LIBNSH_AUTORECONF = YES
+LIBNSH_LICENSE = MIT
+LIBNSH_LICENSE_FILES = LICENSE
+#LIBNSH_CONF_OPTS =
```

Comment:
Please remove. No need to keep "nice to have" settings commented out, not even for reference purposes. We should encourage devs to keep the manual in close reach instead.